api/src/main/java/edu/cornell/mannlib/vitro/webapp/config/ConfigurationPropertiesSmokeTests.java

```diff
@@ -138,17 +174,29 @@ private void checkLanguages(ConfigurationProperties props, StartupStatus ss) {
 					PROPERTY_LANGUAGE_SELECTABLE, selectString));
 		}
 
-		if (buildWithLanguages && !languageFilter) {
+		if (selectableLanguages && !languageFilter) {
 			ss.warning(this, String.format("Problem with Language setup - "
-					+ "build.properties includes one or more additional "
+					+ "languages.selectableLocales in runtime.properties "
+					+ "includes one or more additional "
 					+ "languages (%s = %s), but runtime.properties has "
 					+ "disabled language filtering (%s = %s). This will "
 					+ "likely result in a mix of languages in the "
-					+ "application.", PROPERTY_LANGUAGE_BUILD, buildString,
+					+ "application.", PROPERTY_LANGUAGE_SELECTABLE, selectString,
 					PROPERTY_LANGUAGE_FILTER, filterString));
 		}
 	}
 
+	/** Create a list of the names of available language files. */
+	private List<String> geti18nNames(String i18nBaseDirPath) throws java.io.IOException {
+		List<String> i18nNames = Files.walk(Paths.get(i18nBaseDirPath))
+				.filter(Files::isRegularFile)
+				.map(Path::getFileName)
+				.map(p -> {return p.toString();})
+		    .collect(Collectors.toList());
+
+		return i18nNames;
+	}
+
```

Comment:
Rather than pass the `IOException`, we can wrap it in a `RuntimeException`, so it will not be caught anywhere in this class. The `StartupManager` will catch it, and do the equivalent of `ss.fatal(this, "some simple message", e)`
````
	/** Create a list of the names of available language files. */
	private List<String> geti18nNames(String i18nBaseDirPath) {
		try {
			List<String> i18nNames = Files.walk(Paths.get(i18nBaseDirPath))
					.filter(Files::isRegularFile)
					.map(Path::getFileName)
					.map(p -> {return p.toString();})
					.collect(Collectors.toList());

			return i18nNames;
		} catch (IOException e) {
			throw new RuntimeException("Failed to find language files", e);
		}
	}
````